debian/changelog

```diff
@@ -3,3 +3,7 @@ tensorman (0.1.0) unstable; urgency=medium
   * Initial release.
 
  -- Michael Aaron Murphy <michael@system76.com>  Thu, 19 Sep 2019 4:00:04 -0600
+
+tensorman (0.2.0) unstable; urgency=medium
```

Comment:
Thank you for the contribution! Apologies this took a little while to get to, we're pretty focused on COSMIC stuff right now.

I'm going to test this, but I did see right away that the Debian changelog needs an adjustment-- newer entries should be on the top. Please also use the RFC5322 format for the timestamp. (The `dch` tool should do both of these things for you if you use that to bump the changelog, but otherwise, the adjustments can be made manually.)

Reply:
Thank you for your helpful feedback! I made the changes manually. Please let me know if more adjustments are needed.